### PR TITLE
maintain document order

### PR DESCRIPTION
**Pull request by mdayican**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/BPS-1267

### Change description ###

for now we do not need,
maintain document order

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/resources/db/migration/V057__Add_document_order_column_to_scannable_items.sql [hunk 1]
```diff
@@ -0,0 +1,2 @@
+ALTER TABLE scannable_items
+  ADD COLUMN documentOrder INTEGER DEFAULT 0;
```
Comment (lgonczar) [inline src/main/resources/db/migration/V057__Add_document_order_column_to_scannable_items.sql:2]: Best to add not null contraint.

Comment (mdayican) [reply src/main/resources/db/migration/V057__Add_document_order_column_to_scannable_items.sql:2]: how we can sort out existing ones?

Comment (lgonczar) [reply src/main/resources/db/migration/V057__Add_document_order_column_to_scannable_items.sql:2]: I thought that's why you added the default. Ultimately we shouldn't even have default value for this column, as we'll be always setting it from code.

**Comment by rkondratowicz** [comment]
stale - closing

**Pull request closed without merging**